src/test/java/hudson/plugins/git/GitChangeSetBasicTest.java

```diff
@@ -2,7 +2,10 @@
 
 import java.util.ArrayList;
 import java.util.Arrays;
+
 import static org.junit.Assert.*;
+
+import org.junit.Assert;
```

Comment:
There is already a wildcard import of org.junit.Assert.*, no need to import org.junit.Assert.